### PR TITLE
Add Readme to datasets

### PR DESCRIPTION
**Pull request by oke-aditya**
Adds disclaimer to the dataset folder. Mostly taken from HuggingFace nlp.

### CODE REVIEW

**Comment by oke-aditya** [comment]
Yep. Will add that. Also I will add an example of how to use datasets.

File: mantisshrimp/datasets/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Create a Parser for dataset.
+parser = datasets.pets.parser(data_dir)
+
+# Create a trian and validation dataset
+data_splitter = RandomSplitter([.8, .2])
+
+# Parse the data into train and validation
+train_records, valid_records = parser.parse(data_splitter)
+
+# To geth number of classes
```
Comment (lgvaz) [inline mantisshrimp/datasets/README.md:33]:
```suggestion
# List of all available classes
```

File: mantisshrimp/datasets/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+pet_classes = datasets.pets.CLASSES
+
+```
+Note: - The datasets interface will always have at least two functions: load and parser and an attribute CLASSES
```
Comment (lgvaz) [inline mantisshrimp/datasets/README.md:37]:
```suggestion
Note: - The datasets interface will always have at least two functions: `load` and `parser` and an attribute `CLASSES`
```

File: mantisshrimp/datasets/README.md (hunk 1 above)
Comment (oke-aditya) [reply mantisshrimp/datasets/README.md:33]: Changed

File: mantisshrimp/datasets/README.md (hunk 2 above)
Comment (oke-aditya) [reply mantisshrimp/datasets/README.md:37]: Nice catch. Changed.

**Pull request merged**